{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "7c66dac7",
   "metadata": {},
   "source": [
    "**Main project directory for these analyses is ~/les_complete/public_genomes/LC10-additional-public-les/.**\n",
    "\n",
    "I downloaded all complete and draft genomes from pseudomonas.com to test for STs. I want to see if I find any LES (ST 146 or 683) and whether the 683s specifically group together or in two clades (the deep branching one). I am wondering whether the deep branching clade represents a completely different strain compared to the LES clade.\n",
    "\n",
    "I.e., could the deep branching clade be unrelated and share an ST only due to mutation of one ST to another? Are there any other known 683 isolates that fall in the deep branching clade?\n",
    "\n",
    "```\n",
    "(mlst-env) conrad@conrad-Precision-Tower-3620:~/les_complete/LC10-st-testing/fna-complete/aeruginosa$ for i in *.fna; do echo 'mlst --scheme paeruginosa --legacy --threads 1 '${i}' >> /home/conrad/les_complete/LC10-st-testing/fna-complete/ts-mlst-results.txt' >> ../run-ts-mlst.txt; done\n",
    "```\n",
    "\n",
    "Repeated for draft genomes...  \n",
    "\n",
    "\n",
    "There were 11 ST-683 genomes, including 2 I previously had, so 9 new ST-683 genomes:\n",
    "\n",
    "WH-SGI-V-07405 (fastq available)\n",
    "359\n",
    "5024382738-25\n",
    "5024382738-26\n",
    "19\n",
    "AND07-007 (fastq available)\n",
    "PSA00333 (fastq available)\n",
    "C51\n",
    "mPA08-31\n",
    "\n",
    "Their assemblies are now in /home/conrad/les_complete/public_genomes/LC10-additional-public-les/assemblies/.\n",
    "3 had fastq files available (WH-SGI-V-07405, AND07-007, PSA00333); these are in /home/conrad/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files/.\n",
    "\n",
    "I further downloaded 10 LES genome assemblies from Investigating Pseudomonas aeruginosa population structure and frequency of cross-infection in UK cystic fibrosis clinics - a reference laboratory perspective (Kenna et al. 2023) (all in fastq format):\n",
    "\n",
    "L1_07.21\n",
    "L2_02.20\n",
    "L3_05.19\n",
    "L4_04.21\n",
    "L5_02.21\n",
    "L6_05.21\n",
    "L5_06.19\n",
    "L7_06.21\n",
    "L8_05.18\n",
    "L9_07.19\n",
    "\n",
    "I further downloaded 2 assemblies/fastq from pubmlst that are ST-683. They are not published but appear to be from Australia and collected in 2020:\n",
    "\n",
    "SCHI0002.S.12\n",
    "SCHI0002.S.9\n",
    "\n",
    "\n",
    "#### FastQC on downloaded fastq:\n",
    "```\n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/AND07-007_1.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/AND07-007_2.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/PSA00333_1.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/PSA00333_2.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/WH-SGI-V-07405_1.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ fastqc -t 6 -o raw-fastqc/ raw-fastq-files/WH-SGI-V-07405_2.fastq.gz \n",
    "\n",
    "# kenna list:\n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ for i in $(cat kenna-list.txt); do fastqc -t 6 -o raw-fastqc/ raw-fastq-files/${i}_1.fastq.gz; fastqc -t 6 -o raw-fastqc/ raw-fastq-files/${i}_2.fastq.gz; done\n",
    "\n",
    "# madden 2 genomes:\n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ fastqc -t 6 -o ../raw-fastqc/ SCHI0002.S.9_1.fastq.gz; fastqc -t 6 -o ../raw-fastqc/ SCHI0002.S.9_2.fastq.gz \n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ fastqc -t 6 -o ../raw-fastqc/ SCHI0002.S.12_1.fastq.gz; fastqc -t 6 -o ../raw-fastqc/ SCHI0002.S.12_2.fastq.gz\n",
    "```  \n",
    "\n",
    "#### Trimmomatic on downloaded fastq:\n",
    "```\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ trimmomatic PE -threads 6 AND07-007_1.fastq.gz AND07-007_2.fastq.gz ../trimmed-fastq/AND07-007_1.fastq.gz ../trimmed-fastq/AND07-007_u_1.fastq.gz ../trimmed-fastq/AND07-007_2.fastq.gz ../trimmed-fastq/AND07-007_u_2.fastq.gz ILLUMINACLIP:/home/conrad/software/TruSeq3-PE-2.fa:2:30:10:8:true CROP:300 SLIDINGWINDOW:4:5 MINLEN:31^C\n",
    "\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ trimmomatic PE -threads 6 PSA00333_1.fastq.gz PSA00333_2.fastq.gz ../trimmed-fastq/PSA00333_1.fastq.gz ../trimmed-fastq/PSA00333_u_1.fastq.gz ../trimmed-fastq/PSA00333_2.fastq.gz ../trimmed-fastq/PSA00333_u_2.fastq.gz ILLUMINACLIP:/home/conrad/software/NexteraPE-PE.fa:2:30:10:8:true CROP:150 SLIDINGWINDOW:4:5 MINLEN:31\n",
    "\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ trimmomatic PE -threads 6 WH-SGI-V-07405_1.fastq.gz WH-SGI-V-07405_2.fastq.gz ../trimmed-fastq/WH-SGI-V-07405_1.fastq.gz ../trimmed-fastq/WH-SGI-V-07405_u_1.fastq.gz ../trimmed-fastq/WH-SGI-V-07405_2.fastq.gz ../trimmed-fastq/WH-SGI-V-07405_u_2.fastq.gz ILLUMINACLIP:/home/conrad/software/TruSeq3-PE-2.fa:2:30:10:8:true CROP:150 SLIDINGWINDOW:4:5 MINLEN:31\n",
    "\n",
    "# kenna list:\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ for i in $(cat kenna-list.txt); do trimmomatic PE -threads 6 raw-fastq-files/${i}_1.fastq.gz raw-fastq-files/${i}_2.fastq.gz trimmed-fastq/${i}_1.fastq.gz trimmed-fastq/${i}_u_1.fastq.gz trimmed-fastq/${i}_2.fastq.gz trimmed-fastq/${i}_u_2.fastq.gz CROP:100 SLIDINGWINDOW:4:5 MINLEN:31; done\n",
    "\n",
    "# madden 2 genomes:\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les/raw-fastq-files$ for i in SCHI0002.S.9 SCHI0002.S.12; do trimmomatic PE -threads 6 ${i}_1.fastq.gz ${i}_2.fastq.gz ../trimmed-fastq/${i}_1.fastq.gz ../trimmed-fastq/${i}_u_1.fastq.gz ../trimmed-fastq/${i}_2.fastq.gz ../trimmed-fastq/${i}_u_2.fastq.gz CROP:150 SLIDINGWINDOW:4:5 MINLEN:31; done\n",
    "```  \n",
    "\n",
    "#### Assemble Kenna et al. (2023) genomes:  \n",
    "```\n",
    "(base) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ create_arc_fastqc_trimmomatic_unicycler_jobs.py --analysis unicycler --num_threads 28 --memmax 0 --maxtime 5:00:00 --partitions cpu2017-bf05 --isolate_list kenna-list.txt --chunk_size 1 --fastq_ending .fastq.gz --pair_ending _1,_2 --input_dir /work/parkins_lab/project/conrad/les_complete/public_genomes/LC10-additional-public-les/trimmed-fastq/ --output_dir /work/parkins_lab/project/conrad/les_complete/public_genomes/LC10-additional-public-les/kenna-unicycler/ --jobs_dir kenna-unicycler-jobs/ --prefix knUni683 --env unicycler-env --mode normal --depth_filter 0.25 --min_contig_len 200\n",
    "\n",
    "num_threads_arg = \"28\"\n",
    "memmax_arg = \"0\"\n",
    "maxtime_arg = \"5:00:00\"\n",
    "partitions_arg = \"cpu2017-bf05\"\n",
    "isolate_list_arg = \"kenna-list.txt\"\n",
    "chunk_size_arg = \"1\"\n",
    "fastq_ending_arg = \".fastq.gz\"\n",
    "input_dir_arg = \"/work/parkins_lab/project/conrad/les_complete/public_genomes/LC10-additional-public-les/trimmed-fastq/\"\n",
    "output_dir_arg = \"/work/parkins_lab/project/conrad/les_complete/public_genomes/kenna-unicycler/\"\n",
    "job_prefix_arg = \"knUni683\"\n",
    "jobs_dir_arg = \"kenna-unicycler-jobs/\"\n",
    "mod_arg = \"normal\"\n",
    "depth_filter_arg = \"0.25\"\n",
    "min_contig_len_arg = \"200\"\n",
    "env_arg = \"unicycler-env\"\n",
    "```  \n",
    "\n",
    "#### Simulate reads for assemblies without reads:  \n",
    "```\n",
    "# Isolates requiring simulated reads:\n",
    "#    PA-359\n",
    "#    5024382738-25\n",
    "#    5024382738-26\n",
    "#    PA-19\n",
    "#    C51\n",
    "#    mPA08-31\n",
    "(base) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ for i in $(cat sim-list.txt); do ~/software/art_bin_MountRainier/art_illumina -ss MSv3 -p -i ~/les_complete/public_genomes/LC10-additional-public-les/assemblies/${i}.fasta -l 250 -m 400 -s 110 -f 100 -o raw-fastq-files/${i}_ -rs 42; done\n",
    "```  \n",
    "\n",
    "#### FastQC simulated reads:  \n",
    "```\n",
    "(fastqc-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ for i in $(cat sim-list.txt); do fastqc -t 6 -o raw-fastqc/ raw-fastq-files/${i}_1.fastq.gz; fastqc -t 6 -o raw-fastqc/ raw-fastq-files/${i}_2.fastq.gz; done\n",
    "```  \n",
    "\n",
    "#### Trim simulated reads:  \n",
    "```\n",
    "(trimmomatic-env) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ for i in $(cat sim-list.txt); do trimmomatic PE -threads 6 raw-fastq-files/${i}_1.fastq.gz raw-fastq-files/${i}_2.fastq.gz trimmed-fastq/${i}_1.fastq.gz trimmed-fastq/${i}_u_1.fastq.gz trimmed-fastq/${i}_2.fastq.gz trimmed-fastq/${i}_u_2.fastq.gz CROP:250 SLIDINGWINDOW:4:5 MINLEN:31; done\n",
    "```  \n",
    "\n",
    "**All trimmed fastq files moved to ~/les_complete/public_genomes/public-fastq/trimmed-fastq/.**  \n",
    "\n",
    "#### Annotated all assemblies with Bakta:\n",
    "```\n",
    "(base) conrad@conrad-Precision-Tower-3620:~/les_complete/public_genomes/LC10-additional-public-les$ create_arc_fastqc_trimmomatic_unicycler_jobs.py --analysis bakta --num_threads 40 --memmax 0 --maxtime 05:00:00 --partitions cpu2019-bf05 --isolate_list additional-public-les-list.txt --chunk_size 1 --output_dir /work/parkins_lab/project/conrad/les_complete/public_genomes/LC10-additional-public-les/bakta-unicycler/ --jobs_dir bakta-unicycler-jobs/ --prefix unibkt --env bakta-1.7 --genus Pseudomonas --species aeruginosa --bakta_db /home/conrad.izydorczyk/bakta-db-2023-03-02/db/ --assembly_dir /work/parkins_lab/project/conrad/les_complete/public_genomes/LC10-additional-public-les/assemblies/ --assembly_suffix assembly.fasta --min_contig_len 200\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6600de6c",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Bash",
   "language": "bash",
   "name": "bash"
  },
  "language_info": {
   "codemirror_mode": "shell",
   "file_extension": ".sh",
   "mimetype": "text/x-sh",
   "name": "bash"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
